FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR_GCC/main.c

```diff
@@ -337,19 +335,3 @@ int __write( int iFile,
 
     return iStringLength;
 }
-/*-----------------------------------------------------------*/
-
-void * malloc( size_t size )
```

Comment:
I think this implementation is here to catch accidental calls to malloc. What is the reason to remove this?

Reply:
i think some time ago I had a problem where I needed to disable this, but testing this again everything seems to be ok. So this code should stay in for compatibility reasons.
I'll update this once the above malloc item is resolved.